components/global_onboarding_list/onboarding_checklist.tsx

```diff
@@ -0,0 +1,269 @@
+
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import checklistImg from 'images/onboarding-checklist.svg';
+import {Preferences} from 'utils/constants';
+
+import {TaskListPopover} from './onboarding_checklist_popover';
+import {Task} from './onboarding_checklist_task';
+import Completed from './onboarding_checklist_completed';
+import {CompletedAnimation} from './onboarding_checklist_animations';
+
+const TaskItems = styled.div`
+    border-radius: 4px;
+    border: solid 1px rgba(var(--center-channel-color-rgb), 0.16);
+    background-color: var(--center-channel-bg);
+    max-width: 352px;
+    padding: 1rem 0;
+    transform: scale(0);
+    opacity: 0;
+    box-shadow: 0px 20px 32px rgba(0, 0, 0, 0.12);
+    transition: opacity 150ms ease-in-out 0ms, transform 150ms ease-in-out 0ms;
+    transform-origin: left bottom;
+    height: 518px;
+    max-height: ${document.documentElement.clientHeight}px;
+    overflow-y: auto;
+
+    &.open {
+        transform: scale(1);
+        opacity: 1;
+    }
+
+    h1 {
+        font-size: 20px;
+        padding: 0 24px;
+        margin: 16px 0 0;
+    }
+
+    p {
+        fontSize: 12px;
+        color: rgba(var(--center-channel-color-rgb), 0.72);
+        padding: 4px 24px;
+    }
+
+    .link {
+        font-size: 12px;
+        color: var(--link-color);
+        padding: 12px 24px 0;
+        font-weight: bold;
+        cursor: pointer;
+        display: block;
+    }
+`;
+
+const Button = styled.button<{open: boolean}>(({open}) => {
+    return css`
+        width: 36px;
+        height: 36px;
+        padding: 7px;
+        border-radius: 50%;
+        left: 20px;
+        bottom: 20px;
+        position: fixed;
+        z-index: 21;
+        display: flex;
+        align-items: center;
+        background: var(--center-channel-bg);
+        border: solid 1px rgba(var(--center-channel-color-rgb), 0.16);
+        box-shadow: 0px 6px 14px rgba(0, 0, 0, 0.16);
+        
+        i {
+            color: rgba(var(--center-channel-color-rgb), 0.56);
+        }
+        
+        &:hover {
+            border-color: rgba(var(--center-channel-color-rgb), 0.24);
+            box-shadow: 0px 8px 24px rgba(0, 0, 0, 0.16);
+            
+            i {
+                color: rgba(var(--center-channel-color-rgb), 0.72)
+            }
+        }
+
+        span {
+            width: 20px;
+            height: 16px;
+            background: var(--sidebar-text-active-border);
+            position: fixed;
+            display: ${open ? 'none' : 'block'};
+            border-radius: 12px;
+            color: var(--button-color);
+            font-weight: bold;
+            font-size: 11px;
+            line-height: 16px;
+            bottom: 47px;
+            left: 41px;
+        }
+    `;
+});
+
+const PlayButton = styled.button`
```

Comment:
May be we can add some static width to button,as it is causing issue atleast on my system

Reply:
added    max-width: 175px;